test.js

```diff
@@ -3,7 +3,9 @@ import assert from "node:assert/strict";
 import fs from "node:fs/promises";
 import { fileURLToPath } from "node:url";
 import prettier from "prettier";
-import synchronizedPrettier, { createSynchronizedPrettier } from "./index.cjs";
+import synchronizedPrettier, {
+  createSynchronizedPrettier,
+} from "@prettier/sync";
```

Comment:
This is going too far.

Reply:
Ah this saves us having to duplicate the default `"."` package export for `./index.cjs`

So that Node.js can determine `import` from `require()` entries into the package

Reply:
We don't need type file work internally, I can't see the value.

Reply:
That's fair. I've removed it 😊

You'd have seen this sort of thing appear in the editor:

```mjs
// Module '"@prettier/sync"' has no exported member 'format'
import { format, createSynchronizedPrettier } from "@prettier/sync";
         ^^^^^^
```

But using `require()` in the same `test.js` file would have worked fine:

```mjs
const require = createRequire(import.meta.url)
const { format } = require('./index.cjs')
```